Source/WebCore/dom/Element.cpp

```diff
@@ -1351,7 +1352,15 @@ void Element::scrollTo(const ScrollToOptions& options, ScrollClamping clamping,
     // If the element does not have any associated CSS layout box, the element has no associated scrolling box,
     // or the element has no overflow, terminate these steps.
     CheckedPtr renderer = renderBox();
-    if (!renderer || !renderer->hasNonVisibleOverflow())
+    if (!renderer)
+        return;
+
+    bool rendererCanScroll = [&] {
```

Comment:
Minor preference here for defining the lambda here, and calling it in the `if` condition, rather than calling the lambda inline.